README.md

```diff
@@ -13,13 +13,13 @@ This repository holds community extensions for the [Fyne](https://fyne.io) toolk
 
 This is in early development and more information will appear soon.
 
-## Layouts
```

Comment:
I was about to try and merge this in, but realised this change to README.
A markdown document should have only 1 heading, so the changes here to re-level each item isn't right.